Vault issue 107

nandarshan opened the pull request:
Here, to store secrets, Vault from Hashicorp is being used. This PR implements the getSCPSecret, createSCPSecret, updateSCPSecret, and deleteSCPSecret methods for the SCP in VaultSecretBackend.java. This PR also contains the scripts to start and stop the vault. 

Issue: https://github.com/apache/airavata-mft/issues/107

DImuthuUpe reviewed (changes requested):
Hi @nandarshan this is a great start and thanks for working on this feature. Please refer to this PR in this issue https://github.com/apache/airavata-mft/issues/107 mentioning that you are working on it. 

Regarding the code, remove all standard.out.println and e.printStacktrace as those are not logger friendly. Use a logger function instead. Once you have fixed those, I will do in-depth review of the code

nandarshan commented:
Thanks for the reply @DImuthuUpe. I have removed all println and printStackTrace statements and added logger functions instead. Please let me know if there are any other corrections required.
